apps/passport-server/src/services/generic-issuance/pipelines/PretixPipeline.ts

```diff
@@ -1170,6 +1216,7 @@ export class PretixPipeline implements BasePipeline {
    */
   private async canCheckInForEvent(
     eventId: string,
+    productId: string,
```

Comment:
Presumably this means that there are certain products that can be checked in by some users and not others? I guess the idea is that some checkers can check in my food voucher but would not be able to check in my main event ticket.

This overlaps with the work I'm doing on offline check-in. I don't think it breaks anything but it does make it more complicated.